src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="#" 
+      method="post" 
+      class="form" 
+      data-qa="big"
+  >
+      <img 
+        src="images/Search.svg" 
+        alt="icon" 
+        class="icon"
+      >
+      <input 
+        type="search" 
+        name="biggest" 
+        data-qa="keypress" 
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+    
+    <form 
+      action="#" 
+      method="post" 
+      class="form" 
+      data-qa="small"
+    >
+      <img 
+        src="images/Search.svg" 
+        alt="small-icon" 
+        class="small-icon"
+      >
+      <input 
+        type="search" 
+        name="smaller" 
+        class="small" 
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
```

Comment:
```suggestion
      <img 
        src="images/Search.svg" 
        alt="icon" 
        class="icon"
      >
      <input 
        type="search" 
        name="biggest" 
        data-qa="keypress"
        class="input"
        placeholder="Try &quot;Los Angeles&quot;"
      >
    </form>
    
    <form 
      action="#" 
      method="post" 
      class="form" 
      data-qa="small"
    >
      <img 
        src="images/Search.svg" 
        alt=" icon icon--small" 
        class="small-icon"
      >
      <input 
        type="search" 
        name="smaller" 
        class=" input input--small" 
        data-qa="keypress"
        placeholder="Try &quot;Los Angeles&quot;"
      >
```
Make it work like that 
![image](https://user-images.githubusercontent.com/97547824/206841043-6991b909-4759-4dfd-9c99-a2086e4868cd.png)